packages/react-server/core/TritonAgent/Request.js

```diff
@@ -101,6 +102,27 @@ Request.prototype.type = function (type) {
 	return this;
 }
 
+Request.prototype.noCacheResponse = function () {
+	this._noCacheResponse = true;
+	return this;
+}
+
+Request.prototype.getCacheEntry = function() {
+	if(this._noCacheResponse) {
+		return new CacheEntry(
+			null,
+			this._getCacheAffectingData(),
+			this._cacheWhitelist
+		)
+	} else {
+		return this._cache.entry(
+			this._getCacheAffectingData(),
+			SERVER_SIDE, // createIfMissing
+			this._cacheWhitelist
+		)
+	}
+}
+
```

Comment:
It feels a little weird that calling a public `getCacheEntry` will create an entry in the cache. I feel like maybe `getCacheEntry` should always pass `false` for createIfMissing, and we can leave the call to `this._cache.entry(..., SERVER_SIDE, ...)` in `Response.prototype.end`?